frontend/components/TitleBar.vue

```diff
@@ -1,19 +1,41 @@
 <script setup lang="ts">
 import { onMounted } from 'vue'
 
+const router = useRouter()
+
 interface Props {
   title: string
 }
 const props = withDefaults(defineProps<Props>(), {})
+
+const back = () => { router.go(-1) }
+
 onMounted(() => {
   document.title = 'RMI | PACTA | ' + props.title
 })
 </script>
 
 <template>
-  <h1 class="inline-block text-primary title-bar">
-    {{ title }}
-  </h1>
+  <div class="relative flex flex-column gap-2 align-items-start">
+    <PVButton
+      icon="pi pi-angle-left"
+      class="p-button-secondary p-button-text md:hidden p-0 text-600 pr-2"
+      style="margin-left: -.25rem"
```

Comment:
why not use negative PrimeFlex classes here and below? The classes encourage consistent visual shenanigans. For example, [`-ml-1` exactly matches this style](https://www.primefaces.org/primeflex/margin), AFAICT

Reply:
TIL!!! Thank you! I had no idea they had negative margins in primeflex! Thanks!